middle_end/base_types/compilation_unit.ml

```diff
@@ -71,3 +72,13 @@ let get_current_exn () =
   | Some current -> current
   | None -> Misc.fatal_error "Compilation_unit.get_current_exn"
 let get_current_id_exn () = get_persistent_ident (get_current_exn ())
+
+let startup =
+  lazy ({
+    id = Ident.create_persistent "*startup*";
+    linkage_name = Linkage_name.create "caml_startup";
+    hash = Hashtbl.hash "*startup*";
+    is_startup = true;
+  })
```

Comment:
This is not used.
By the way, note that Ident.create_persistent does not increment ident counter, so it does not need to be delayed by a lazy.